app/controllers/actions/IdentifierAction.scala

```diff
@@ -43,10 +46,18 @@ class AuthenticatedIdentifierAction @Inject()(override val authConnector: AuthCo
       Redirect(routes.UnauthorisedController.onPageLoad())
     }
 
-    authorise().retrieve(Retrievals.internalId) {
-      _.map {
-        internalId => block(IdentifierRequest(request, internalId))
-      }.getOrElse(throw new UnauthorizedException("Unable to retrieve internal Id"))
+    // TODO: remove fake EORI number when in public beta
+    def eoriNumber(enrolments: Enrolments): String = {
+      enrolments.getEnrolment(cdsEnrolment).flatMap(_.getIdentifier("EORINumber")) match {
+        case Some(eori) => eori.value
+        case _ if config.isCdsEnrolmentCheckEnabled => throw InsufficientEnrolments()
+        case _ => "GB6723190" // fake EORI number
```

Comment:
why do we need this fake hard codes value now?

Reply:
Because in the private beta phase we want to allow users without a EORI number to use the application